bot/exts/holidays/valentines/be_my_valentine.py

```diff
@@ -73,12 +98,33 @@ async def send_valentine(
         channel = self.bot.get_channel(Channels.sir_lancebot_playground)
         valentine, title = self.valentine_check(valentine_type)
 
-        embed = discord.Embed(
-            title=f"{emoji_1} {title} {user.display_name} {emoji_2}",
-            description=f"{valentine} \n **{emoji_2}From {ctx.author}{emoji_1}**",
-            color=Colours.pink
-        )
-        await channel.send(user.mention, embed=embed)
+        if anon.lower() == "anon":
+            embed = discord.Embed(
+                title=f"{emoji_1} {title} {user.display_name} {emoji_2}",
+                description=f"{valentine} \n **{emoji_2}From an anonymous admirer{emoji_1}**",
+                color=Colours.pink
+            )
+        
+        else:
+            embed = discord.Embed(
+                title=f"{emoji_1} {title} {user.display_name} {emoji_2}",
+                description=f"{valentine} \n **{emoji_2}From {ctx.author}{emoji_1}**",
+                color=Colours.pink
+            )
```

Comment:
There is unnecessary duplication here, you could just have something like
```python
from_user = "an anonymous admirer" if anon.lower() == "anon" else ctx.author
```
and format that into a single embed call.